x/authz/codec.go

```diff
@@ -22,3 +35,8 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 
 	msgservice.RegisterMsgServiceDesc(registry, MsgServiceDesc())
 }
+func init() {
+	// Register all Amino interfaces and concrete types on the global Amino codec so that this can later be
+	// used to properly serialize MsgGrant and MsgExec instances
+	RegisterLegacyAminoCodec(legacy.Cdc)
```

Comment:
We might need to add some docs somewhere so developers know they need to do the same for each of their modules in order to grant that `MsgExec` will be serialized properly.

Reply:
Could you add these docs as part of this PR? I think a small paragraph in `docs/core/encoding.md`, there's an `### Amino` section

Reply:
Added a subsection called _Authz authorizations_